packages/elements-core/src/components/TryIt/Mocking/useMockingOptions.ts

```diff
@@ -2,5 +2,5 @@ import { atom, useAtom } from 'jotai';
 
 import { MockingOptions } from './mocking-utils';
 
-const persistedMockingOptionsAtom = atom<MockingOptions>({ isEnabled: false });
+const persistedMockingOptionsAtom = atom<MockingOptions>({});
```

Comment:
Very tiny thing, but it bothers me.

I would remove `persisted` prefix from here.

I believe that in other parts of the codebase we refer to things as being "persisted" if they are saved somewhere between sessions (like local storage).

For atoms we just use `somethingSomethingAtom` convention, so `mockingOptionsAtom` is a perfect name IMHO.

Reply:
Completely agree! It confused me as well, I just didn't add these so wasn't sure re making name change. Will do though.